frontend/src/Routings.tsx

```diff
@@ -13,6 +15,8 @@ function Routings() {
         <Route path="/" element={<RoomPages />} />
         <Route path="/user/:id" element={<UserDetail />} />
         <Route path="/room/:id" element={<RoomPage />} />
+        <Route path="/user/change_email" element={<ChangeUserEmail />} />
```

Comment:
you should name url paths with `camelCase` instead of `snake_case`

`change_email` => `changeEmail`